translate/storage/php.py

```diff
@@ -285,8 +285,10 @@ def getoutput(self, indent="", name=None):
             fmt = "{1}{2}{1},\n"
         elif "->" in self.name:
             fmt = "{0} => {1}{2}{1},\n"
-            if name is not None:
-                name = name[0] + phpencode(name[1:-1], name[0]) + name[-1]
+            if name is not None and name[0] == name[-1] == self.escape_type:
```

Comment:
This will break for strings having `self.escape_type == '"'` because the name has always single quote now. I'm not sure it's a good idea to use escape_type as that is the escape type detected for the value, not for the key (I've already mentioned this in https://github.com/translate/translate/pull/5350).

Reply:
so should i change it to name[0] then?